x/wasm/keeper/keeper.go

```diff
@@ -584,14 +585,15 @@ func (k Keeper) callMigrateEntrypoint(
 	querier := k.newQueryHandler(sdkCtx, contractAddress)
 
 	prefixStoreKey := types.GetContractStorePrefix(contractAddress)
-	vmStore := types.NewStoreAdapter(prefix.NewStore(runtime.KVStoreAdapter(k.storeService.OpenKVStore(sdkCtx)), prefixStoreKey))
+	vmStore := prefix.NewStore(runtime.KVStoreAdapter(k.storeService.OpenKVStore(sdkCtx)), prefixStoreKey)
```

Comment:
This should fix the tests imo

```suggestion
	vmStore := runtime.KVStoreAdapter(k.storeService.OpenKVStore(sdkCtx))
```